src/test/java/org/example/BaseTest.java

```diff
@@ -0,0 +1,42 @@
+package org.example;
+
+
+import com.networknt.schema.*;
+import org.testng.annotations.BeforeSuite;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public abstract class BaseTest {
+
+    Properties loadProperties;
+
+    protected String token;
+
+    @BeforeSuite
+    public void loadProps() {
+        ConfigProvider configProvider = new ConfigProvider();
+        loadProperties = configProvider.loadConfig();
+        token = loadProperties.getProperty("token");
+    }
+
+
+    public List<ValidationMessage> validatePayload(String payload, String schemaName) {
```

Comment:
Keep your JSON Schemas in src/test/resources/schemas/, and load them dynamically by template name:
create class JsonValidator and move logic there